package.json

```diff
@@ -24,13 +25,15 @@
     "sax": "^1.1.3"
   },
   "devDependencies": {
+    "@types/node": "^7.0.12",
     "assert-diff": "^1.0.1",
     "istanbul": "*",
     "mocha": "*",
     "muk-prop": "^1.0.0",
     "nock": "*",
     "sinon": "^2.0.0",
-    "stream-equal": "~1.0.0"
+    "stream-equal": "~1.0.0",
+    "typescript": "^2.2.2"
```

Comment:
Does this need to be included? In another recent similar PR (for another project), `typescript` was not part of `devDependencies`.

Which does make sense since someone working with TS would already have that on their project.